mockito-kotlin/src/main/kotlin/org/mockito/kotlin/Matchers.kt

```diff
@@ -29,6 +29,7 @@ import org.mockito.ArgumentMatcher
 import org.mockito.ArgumentMatchers
 import org.mockito.kotlin.internal.createInstance
 import kotlin.reflect.KClass
+import kotlin.reflect.jvm.internal.impl.load.kotlin.JvmType.Object
```

Comment:
I don't see a usage of this class in this file. Is this import necessary and the additional dependency?

Reply:
You were right that the import was unnecessary. I removed it.

The dependency unfortunately still is. The dependency enables reflection for Kotlin, which is necessary for detecting and introspecting value classes.